src/main/scala/com.snowplowanalytics.sauna/observers/AmazonS3Observer.scala

```diff
@@ -53,7 +54,7 @@ class AmazonS3Observer(s3: S3, sqs: SQS, queue: Queue) extends Actor with Observ
 
```

Comment:
I'm still not sure if this should be implemented - observers are supposed to support multiple instances of a single type as far as I know.